app/gui/qt/lang/sonic-pi_fr.ts

```diff
@@ -571,7 +570,7 @@ Veuillez nous excuser, une erreur critique s&apos;est produite pendant le démar
     <message>
         <location filename="../mainwindow.cpp" line="2000"/>
         <source>Run</source>
-        <translation>Run</translation>
+        <translation>Lancer</translation>
```

Comment:
If we rename the 'Run' button to 'Lancer' (if this is indeed the button label) we should also adapt the tutorial (where it is referenced more than twenty times).